pkg/registry/rbac/validation/rule.go

```diff
@@ -361,6 +384,18 @@ func (r *StaticRoles) GetRole(namespace, name string) (*rbacv1.Role, error) {
 	return nil, errors.New("role not found")
 }
 
+func (r *StaticRoles) GetRoleWithMultiTenancy(namespace, tenant, name string) (*rbacv1.Role, error) {
```

Comment:
Non-critical: it is better to arrange the param in the order of tenant/ns/name to show the hierarchy of these concepts

Reply:
fixed